llvm/test/CodeGen/AMDGPU/sdwa-peephole-reg-sequence.mir

```diff
@@ -0,0 +1,55 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900  -stop-after=si-peephole-sdwa -o - %s | FileCheck %s
```

Comment:
I don't understand what this run line is doing, and it should probably be an llc error. Where is this starting? I'm suspecting it's forming the default empty function, codegenning that, and then it somehow merges with whatever MIR happened to be there from the initial parse?

Reply:
Thanks, fixed it. I am also not sure what it really does.